apps/web/src/lang/fr.json

```diff
@@ -6161,6 +6185,10 @@
     "defaultMessage": "Courriel du gouvernement",
     "description": "Title for the government email block in the manager info section of the single search request view."
   },
+  "d45Ihy": {
+    "defaultMessage": "Prochainement",
```

Comment:
`q5FQbu`, Coming Soon! is translated as _À venir!_. 
Which one should be used @gobyrne?